src/RepoAuditor/Plugins/CommunityStandards/Requirements/Citation.py

```diff
@@ -0,0 +1,64 @@
+# -------------------------------------------------------------------------------
+# |
+# |  Copyright (c) 2024 Scientific Software Engineering Center at Georgia Tech
+# |  Distributed under the MIT License.
+# |
+# -------------------------------------------------------------------------------
+"""Contains the Citation requirement."""
+
+import textwrap
+
+from RepoAuditor.Plugins.CommunityStandards.Impl.ExistsRequirementImpl import (
+    ExistsRequirementImpl,
+)
+
+
+class Citation(ExistsRequirementImpl):
+    """Validates that a CITATION file is present which is used to make the repository citable.
+
+    Details of how GitHub looks for citation files can be found here: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files
+    """
+
+    def __init__(self) -> None:
```

Comment:
This should be a requirement that users opt in to (rather than enabled by default), because it doesn't apply to all git repositories (as an example, it is not an error if `RepoAuditor` doesn't have a citation file).

There should be a setting for this in a base class `__init__` method.

Reply:
Good call. Updated!